server/chat-plugins/datasearch.ts

```diff
@@ -806,17 +800,9 @@ function runDexsearch(target: string, cmd: string, canAll: boolean, message: str
 			}
 
 			if (target === 'pivot') {
-				let includeBatonPass = false;
-				if (parameters.length > 1) {
-					if (parameters[1].trim().toLowerCase() === 'batonpass') {
-						includeBatonPass = true;
-					} else {
-						return {error: "The parameter 'pivot' cannot have alternative parameters other than 'batonpass'."};
-					}
-				}
 				for (const move in Dex.data.Moves) {
 					const moveData = Dex.getMove(move);
-					if (moveData.selfSwitch && (includeBatonPass || moveData.id !== 'batonpass')) {
```

Comment:
why are you removing the baton pass parameter?

Reply:
That's the way the pivot parameter already works. If we're going to include baton pass that's fine, but the decision to not include it wasn't mine.

Reply:
The reasoning at https://www.smogon.com/forums/threads/make-pivot-moves-a-ms-category.3672772/ is that most formats ban Baton Pass, so if BP was included, searching by pivot would have little value.